shared/fs/row/drop-target.native.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import {Box} from '../../common-adapters'
+
+const DropTarget = Box
+
+export default DropTarget
```

Comment:
Could merge this and the desktop, but just export Box if `isMobile`. So we don't have to have a `.flow` file here.

Reply:
Done.